R/check_result.R

```diff
@@ -5,19 +5,17 @@
 #' If the user result exactly matches a known \code{result}, \code{check_result}
 #' returns the matching message value.
 #'
-#' @param results A \code{\link{results}} object that contains possible \code{\link{result}} values to compare against.
 #' @param correct A character string to display if the student answer matches
 #'   a known answer.
 #'   This character string will be run through \code{glue::\link[glue]{glue_data}} with \code{list(correct = TRUE, message = "<result message>")}. where message is the matched result message.
```

Comment:
`matched` should be talked about here.

Reply:
added `#' @template matched` to line 8 of the file